src/UI/App.jsx

```diff
@@ -179,44 +179,21 @@ export default class App extends React.Component {
 	 * @param deviceVersion
 	 * @param referer
 	 * @param authorization
+	 * @param force bool If true, it will not check for any existing registrations and force a new one
 	 */
 	subscribeDevice = (
 		pushToken, pushType, pushEnabled,
 		userAdditionalInformation, type, deviceVersion,
-		referer, authorization,
+		referer, authorization, force,
 	) => {
-		const body = {
-			pushToken,
-			pushType,
-			pushEnabled,
-			userAdditionalInformation,
-			type,
-			version: deviceVersion,
-			referer,
-			authorization,
-		};
-
-		const storeIntoLocalStorage = JSON.stringify({
-			...body,
-			accountIdentification: this.Api.accountIdentification,
-			deviceIdentification: this.Api.deviceIdentification,
-		});
-
-		// Check registration in local storage
-		const storedDeviceInformation = localStorage.getItem("deviceInformation");
-		if(storedDeviceInformation === storeIntoLocalStorage) {
-			// Create cookie if it doesn't exist already (and the persistDeviceBetweenDomain setting is used)
-			if(this.state.persistDeviceBetweenDomain && !this.getDeviceIdentificationCookie()) {
-				this.createDeviceIdentificationCookie(
-					this.state.deviceIdentification,
-					this.state.persistDeviceBetweenDomain,
-				);
-			}
-
-			Logger.debug("Found same device information from localStorage, using that instead of registering a new device");
-			this.Api.deviceRegistered = true;
-			return; // No need to call the API if we don't have any new data
+		if(this.Api.deviceRegistered && !force) {
+			Logger.debug("Device is already registered, not registering a new one");
+			return; // Don't register if we already are registered
```

Comment:
als je al een subscription hebt wordt data niet leeg gemaakt?

Reply:
https://tracebuzz-hub.slack.com/archives/C9P8904JH/p1684929076454449?thread_ts=1684918432.094339&cid=C9P8904JH
Is voor deze issue niet besproken